bin/bovine.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+import os
+import yaml
+##########################
+# Read the Static Directory
+##########################
+
+
+class Inventory:
+  def __init__(self, root_directory):
+    # get the files in the root_directory
+    # self.
+    self.root_directory = root_directory
+    # import the yaml from the files that we found above
+  def list_files(self, root_directory):
+    directory_files = os.walk(root_directory)
+    filenames = []
+    for root, dirs, files in directory_files:
+      for name in files:
+        filenames.append(os.path.join(root, name))
+    return filenames
+    # print(filenames)
+
+
+  def import_yaml(self, filenames):
+    load_of_yaml = []
+    for name in filenames:
+      with open(name, 'r') as f:
+        load_of_yaml = yaml.load(f)
+    return load_of_yaml
+
+test_inventory = Inventory('../test/test_data')
```

Comment:
I think I should leave a brief comment about var naming.  

If I were to see the var 'test_inventory' referenced later, I would assume it has some actual inventory.  Instead, this var is just a file path+name.  

I would highly recommend calling anything that looks like that something_file... e.g. test_inventory_file.